ci/staging/migrate-database.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+APP_NAME="phoenix-starter"
+
+# Get service networkConfiguration
+AWSVPC_CONFIG=$(aws ecs describe-services --services "${APP_NAME}" --cluster "${APP_NAME}" | jq -c ".services[] | select( .serviceName | contains(\"$APP_NAME\") ) | .networkConfiguration.awsvpcConfiguration" | tr -d \" | tr ':' '=')
+
+aws ecs run-task --cluster "${APP_NAME}" --task-definition phoenix-starter-migrate --network-configuration "awsvpcConfiguration=${AWSVPC_CONFIG}"
```

Comment:
When you run this, is the output just a JSON object from the API (that is what I recall)? Wondering if we could capture the log and redirect that to STDOUT so you can see output without having to go into Cloudwatch? (This is an enhancement, not a fixup to this PR.)